translations/fr.json

```diff
@@ -699,22 +671,21 @@
   "gamestatus.destroyed": "détruite(s)",
   "gamestatus.tasks": "Tâches {0}/{1}",
   "gamestatus.tasks.complete": "Complétée!",
-
   "settings.title": "PARAMÈTRES",
   "settings.volume": "Volume sonore {0}",
   "settings.soundeffects": "Effets sonores",
   "settings.ambience": "Ambiance",
-  "settings.tribemusic": "Son de tribu",
-  "settings.suggestions": "Suggestions de jeu",
+  "settings.tribemusic": "Musique de tribu",
+  "settings.suggestions": "Suggestions",
   "settings.info": "Information sur les constructions",
   "settings.confirm": "Confirmer la fin du tour",
-  "settings.saveexit": "SAUVEGARDER \n\u0026\n QUITTER",
+  "settings.saveexit": "SAUVEGARDER \u0026 QUITTER",
   "settings.on": "ON",
   "settings.off": "OFF",
-  "settings.language": "Langage",
+  "settings.language": "Language",
```

Comment:
In french, there is no "u" in "langage", it's like in the word "garage" unlike in english where language takes an "u"